runtime/float.go

```diff
@@ -260,7 +261,8 @@ func floatRDivMod(f *Frame, v, w *Object) (*Object, *BaseException) {
 }
 
 func floatRepr(f *Frame, o *Object) (*Object, *BaseException) {
-	return NewStr(strconv.FormatFloat(toFloatUnsafe(o).Value(), 'g', -1, 64)).ToObject(), nil
+	v := unsignPositiveInf(strings.ToLower(strconv.FormatFloat(toFloatUnsafe(o).Value(), 'f', -1, 64)))
+	return NewStr(v).ToObject(), nil
```

Comment:
This causes other problems :(

From Travis: AssertionError: u'1e+06' != '1000000'